pkgs/top-level/all-packages.nix

```diff
@@ -8569,13 +8588,15 @@ with pkgs;
       limesuite
       soapyairspy
       soapyaudio
-      soapybladerf
       soapyhackrf
-      soapyplutosdr
       soapyremote
       soapyrtlsdr
+    ]
+    ++ (lib.optionals stdenv.hostPlatform.isLinux [
+      soapybladerf
```

Comment:
We can fix soapybladerf build on darwin by fixing libbladeRF:
```patch
--- i/pkgs/by-name/li/libbladeRF/package.nix
+++ w/pkgs/by-name/li/libbladeRF/package.nix
@@ -62,7 +62,7 @@ stdenv.mkDerivation rec {
   ];

   env = lib.optionalAttrs stdenv.cc.isClang {
-    NIX_CFLAGS_COMPILE = "-Wno-error=unused-but-set-variable";
+    NIX_CFLAGS_COMPILE = "-Wno-error=unused-but-set-variable -Wno-error=tautological-overlap-compare";
   };

   hardeningDisable = [ "fortify" ];
```

Reply:
Added as a separate commit to this PR.